src/scss/3_components/form/field-tags.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../init';
+
+.field-name-field-tags {
+  margin: 0 0 1rem;
+
+  a {
+    @include adjust-font-size-to(12px);
+    display: inline-block;
+    background: url('../../../images/tag.png') no-repeat;
```

Comment:
Use resolve() to reference images and the postcss hook will take care of the relativity of the file etc. See nidirect_theme for examples